plugins/content/transcript/lib/api/filters/KalturaTranscriptAssetFilter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaTranscriptAssetFilter extends KalturaAttachmentAssetFilter
+{	
+	/**
+	 * @dynamicType KalturaAssetType
+	 * @var string
+	 */
+	public $typeIn;
+
+	public function __construct()
+	{
+		$this->typeIn = TranscriptPlugin::getAssetTypeCoreValue(TranscriptAssetType::TRANSCRIPT);
```

Comment:
Please do the same also in __wakeup in case it will be serialized in a response-profile.